language/it/info_acp_donation

```diff
@@ -0,0 +1,340 @@
+	'PPDE_DROPBOX_VALUE'                   => 'menu a tendina valore donazione',
+	'PPDE_DROPBOX_VALUE_EXPLAIN'           => 'Imposta i numeri che si desiderano visualizzare nella lista a tendina.<br />Usa <strong>la virgola</strong> (",") <strong>senza spazio</strong> per separare ciascun valore.',
+
+	// PayPal IPN settings
+	'PPDE_LEGEND_IPN_SETTINGS'             => 'PayPal IPN settings',
+	'PPDE_LEGEND_IPN_PPDE_IPN_AUTOGROUP'   => 'Auto-group',
+	'PPDE_IPN_AG_ENABLE'                   => 'Abilita Auto Group',
+	'PPDE_IPN_AG_ENABLE_EXPLAIN'           => 'Permette di aggiungere i donatori a un gruppo predefinito.',
+	'PPDE_IPN_AG_DONORS_GROUP'             => 'Gruppo Donatori',
+	'PPDE_IPN_AG_DONORS_GROUP_EXPLAIN'     => 'Selezionare il gruppo che ospiterà i membri donatori.',
+	'PPDE_IPN_AG_GROUP_AS_DEFAULT'         => 'Imposta gruppo donatori come predefinito',
+	'PPDE_IPN_AG_GROUP_AS_DEFAULT_EXPLAIN' => 'Abilitalo per impostare il gruppo donatori come gruppo predefinito per gli utenti che hanno fatto una donazione.',
+	'PPDE_IPN_ENABLE'                      => 'Abilita IPN',
+	'PPDE_IPN_ENABLE_EXPLAIN'              => 'Attivare questa opzione se si desidera utilizzare notifica immediata del pagamento dei servizi PayPal.<br />Se si abilita, tutte le caratteristiche che dipendono da PayPal IPN saranno disponibili sotto.',
+	'PPDE_IPN_LOGGING'                     => 'Abilita log errori',
+	'PPDE_IPN_LOGGING_EXPLAIN'             => 'Log errori e dati da PayPal IPN in <strong>/store/ppde_transactions.log</strong>.',
+
+	// PayPal sandbox settings
+	'PPDE_LEGEND_SANDBOX_SETTINGS'         => 'PayPal sandbox settings',
+	'PPDE_SANDBOX_ENABLE'                  => 'Sandbox testing',
+	'PPDE_SANDBOX_ENABLE_EXPLAIN'          => 'Attivare questa opzione se si desidera utilizzare PayPal Sandbox al posto dei servizi PayPal.<br />Utile per sviluppatori / tester. Tutte le operazioni sono fittizie.',
+	'PPDE_SANDBOX_FOUNDER_ENABLE'          => 'Sandbox solo per i fondatori',
+	'PPDE_SANDBOX_FOUNDER_ENABLE_EXPLAIN'  => 'Se abilitato, Sandbox PayPal verrà visualizzato solo dai fondatori della board.',
+	'PPDE_SANDBOX_ADDRESS'                 => 'Indirizzo PayPal sandbox',
+	'PPDE_SANDBOX_ADDRESS_EXPLAIN'         => 'Imposta qui il tuo indirizzo e-mail PayPal Sandbox.',
+
+	// Stats Donation settings
+	'PPDE_LEGEND_STATS_SETTINGS'           => 'Statistiche configurazione donazione',
+	'PPDE_STATS_INDEX_ENABLE'              => 'Statistiche di visualizzazione donazione su in indice',
+	'PPDE_STATS_INDEX_ENABLE_EXPLAIN'      => 'Attivare questa opzione se si desidera visualizzare le statistiche di donazione sull\'indice.',
+	'PPDE_RAISED'                          => 'donazione in sospeso',
+	'PPDE_RAISED_EXPLAIN'                  => 'Importo  corrente raccolto con le donazioni.',
+	'PPDE_GOAL'                            => 'Obiettivo donazione',
+	'PPDE_GOAL_EXPLAIN'                    => 'L\'importo totale che si desidera raggiungere.',
```

Comment:
Hi,
Please don't use `\'` but `’`.
You can find this chars on line 35 of this file ;)